### ISSUE TITLE
change_if_to_case: false positive if methods are used

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#2771]
```abap
REPORT zfoobar.
IF is_tool( ) = abap_true.
"...
ELSEIF is_pass( ) = abap_true.
"...
ENDIF.
```

`/zfoobar.prog.abap[2, 1]: Change IF to CASE(change_if_to_case)`

**Comment by larshp** [comment]
hmm, it can be expressed as

```abap
case abap_true.
  when is_tool( ).
  when is_pass( ).
endcase.
```

but however, this is a boolean compare, so it will not be reported anyhow, after https://github.com/abaplint/abaplint/pull/2809 is merged

**Comment by mbtools** [comment]
👍
